ledger/participant-integration-api/src/main/scala/db/migration/postgres/V32_1__Fix_key_hashes.scala

```diff
@@ -25,8 +27,11 @@ private[migration] final class V32_1__Fix_key_hashes extends BaseJavaMigration {
       fixHash = conn.prepareStatement(FIX_HASH)
 
       selectKeys = conn.createStatement()
+      conn.setFetchSize(BATCH_SIZE)
```

Comment:
I think you meant to use `selectKeys.setFetchSize` here.

Reply:
yeah, thanks. 🤦